docs/gallery/articles/supply_chain_monitoring/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Supply Chain Monitoring
+category: decision_support
+data-keywords: dashboard maps enterprise
+short-description: Supply Chain Monitoring dashboard presented as a simulation.
+img: supply_chain_monitoring/images/home_page.png
```

Comment:
You have to put an `order` to your article and it should correspond where it should be seen in the index pages and it should be coherent on where it is in the navigation too

Reply:
Ah, I used the `supply_chain` article as a reference which didn't have that metadata. I don't really understand what you mean though. Can you give me an example? E.g. `supply_chain_3_echelons` has order=3, but I don't understand how this affects its position

Reply:
The order is the order that the demos are appearing in the index pages with their cards. Supply chain 3 echelon should be the third in the index page: 
![image](https://github.com/user-attachments/assets/d769668b-c948-4b78-9fc7-bcb9828d7c39)


It seems that he is 4 but because supply-chain is not correct and has no order. (which is a bug bu having it first is correct for supply chain)

Reply:
Okay, I think I understand. So since the current final article (tweet_generation) is order=23, I can put this as order=24? Or alternatively, if I want to insert it somewhere, I need to increment the order for every article after it?

Let's say I set this article as order=24, what am I supposed to change in `mkdocs.yml_template`?

Reply:
You should increment the order of every article because I guess we want it to appear in the middle or top of the index when you go to Gallery.